docs/troubleshooting.md

```diff
@@ -177,37 +177,6 @@ Or:
 $ HOROVOD_GPU_ALLREDUCE=NCCL HOROVOD_NCCL_INCLUDE=/path/to/nccl/include HOROVOD_NCCL_LIB=/path/to/nccl/lib pip install --no-cache-dir horovod
 ```
 
-### NCCL 2 is not found during runtime
```

Comment:
Due to the fact that we now statically link NCCL 2 into Horovod during installation, this issue can no longer happen.